### ISSUE TITLE
 Email test or email after activation does not send!

### ISSUE THREAD

**Issue by Sergio1981** [rainlab/user-plugin#207]
[SOLVED]
Hi friend. 
I am trying to send test email and the email does not arrive in my mailbox. Put to activate the manual form of User to the email would be sent to him warning activation. In neither case do I get the email be sent . I am using the latest version of the plugin . Please need a solution ! Thank you very much.

**Comment by Sergio1981** [comment]
When I sus Sendmail I get the error message with : Undefined variable: pipe.

**Comment by daftspunk** [comment]
This is probably more due to the October system than the User plugin specifically, further the error appears to be related to your web server configuration (perhaps sendmail is not installed?). In most cases it is common to use a third party provider like Amazon SES or Mailgun.

**Comment by Sergio1981** [comment]
First I want to praise you . I really like the system and I've done videos in my channel buzzing! Two days ago I was sending normally. When I updated the User plugin and went to do a final test found the problem . I was using PHPmail and now not with him I send . Again installed the whole system and nothing happened. Told to support and they said the PHPmail is properly configured . I found it strange to be working and stop
